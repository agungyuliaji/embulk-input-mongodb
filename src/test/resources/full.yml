in:
  type: mongodb
  uri: mongodb://localhost:27017/my_database
  collection: "my_collection"
  fields:
    - { name: id, type: string }
    - { name: name, type: string }
    - { name: rank, type: long }
  query: '{ rank: { $gte: 3 } }'
  sort: '{ rank: -1 }'
out:
  type: file
  path_prefix: ./tmp/full
  file_ext: csv
  formatter:
    type: csv
    header_line: true
    charset: UTF-8
    newline: CRLF