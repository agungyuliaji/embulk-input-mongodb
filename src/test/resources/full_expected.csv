id,name,rank
55eae883689a08361045d652,obj9,9
55eae883689a08361045d651,obj8,8
55eae883689a08361045d650,obj7,7
55eae883689a08361045d64f,obj6,6
55eae883689a08361045d64e,obj5,5
55eae883689a08361045d64d,obj4,4
55eae883689a08361045d64c,obj3,3
